source/bear/source/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
-include(FindPythonInterp)
-if (PYTHONINTERP_FOUND)
-    set(BEAR_PYTHON_EXECUTABLE "python${PYTHON_VERSION_MAJOR}")
```

Comment:
Was thinking to use `PYTHON_EXECUTABLE` (which has the full path of the interpreter) and `basename` that string. Or maybe CMake can do return the executable name from a path?